pkg/plugins/scms/github/pullrequest.go

```diff
@@ -330,9 +330,9 @@ func (p *PullRequest) EnablePullRequestAutoMerge() error {
 
 	if p.spec.UseTitleForAutoMerge {
 		if strings.EqualFold(p.spec.MergeMethod, "squash") {
-			input.CommitHeadline = githubv4.NewString(githubv4.String(fmt.Sprintf("%s (#%s)", p.spec.Title, p.remotePullRequest.ID)))
+			input.CommitHeadline = githubv4.NewString(githubv4.String(fmt.Sprintf("%s (#%d)", p.Title, p.remotePullRequest.Number)))
```

Comment:
Something odd is going on however.

`p.spec.Title` being empty... 😅 yet `p.spec.UseTitleForAutoMerge` being correctly set to true.

Reply:
nvm, it makes sense. Not sure what title is used for in the spec though when it is on the main pull request target.

Reply:
The question is whether this is a bug or not.

I don't know the history why we have both `p.Title` and `p.Spec.Title`
However looking at the code `p.spec.Title` is only used inside: `GetChangelogTitle`

Not sure what is meant with that seperation 🤔

I guess by `p.Title` I have complete control and where `p.Spec.Title` would prepend `[updatecli]`

Not sure why the manifest allows for both flows.

Reply:
This works
```yaml
pullrequests:
  jenkins:
    title: Bump jenkins.version from 2.319.3 to {{ source "jenkins" }} for bom-2.319.x
    kind: github
    scmid: github
    targets:
    - jenkins
    spec:
      labels:
      - dependencies
      automerge: true
      mergemethod: squash
      usetitleforautomerge: true
```

Also works but prepends `[updatecli]`
```
pullrequests:
  jenkins:
    kind: github
    scmid: github
    targets:
    - jenkins
    spec:
      title: Bump jenkins.version from 2.319.3 to {{ source "jenkins" }} for bom-2.319.x
      labels:
      - dependencies
      automerge: true
      mergemethod: squash
      usetitleforautomerge: true
```